sway-lsp/src/core/session.rs

```diff
@@ -127,7 +129,10 @@ impl Session {
     // update sway config
     pub fn update_config(&self, options: Value) {
```

Comment:
We can remove this function entirely as we aren't allowing users to update the config from the plugin anymore.